styles.scc

```diff
@@ -0,0 +1,32 @@
+#image {
+    display: block;
+    max-width: 100%;
+    
```

Comment:
Kindly check for empty rows and delete them. This makes your code more readable and clean 🤙🏻

Reply:
Kudos for pointing out the issue and explaining the why.💯